daemon/src/maker_cfd.rs

```diff
@@ -551,7 +551,7 @@ impl Actor {
     async fn handle_oracle_attestation(&mut self, attestation: oracle::Attestation) -> Result<()> {
         tracing::debug!(
             "Learnt latest oracle attestation for event: {}",
-            attestation.id
+            attestation.id.0
```

Comment:
In these scenarios, I'd favor implementing `fmt::Display` instead of reaching into the newtype. Takes 10 seconds and moves into the right direction. If you don't want to write it by hand, also happy to use something like https://jeltef.github.io/derive_more/derive_more/display.html although more macros means longer compile times in the long run.